docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -132,14 +137,16 @@ options:
       - title: Upgrading From Scala 2.11
         url: chisel3/docs/appendix/upgrading-from-scala-2-11.html
```

Comment:
```suggestion
      - title: Upgrading From Scala 2.11
        url: chisel3/docs/appendix/upgrading-from-scala-2-11.html
      - title: Frequently Asked Questions
        url: chisel3/docs/appendix/faqs.html        
```

Reply:
So, i left off the FAQ because I couldn't get the "how to get verilog/firrtl" instructions to actually work. I am happy to spend more time on those or remove that from the FAQ. So yes, let's take this change here but then I need to clean up the FAQ in the other PR

Reply:
I think it's okay to leave off the FAQ and get your other changes merged, they're a huge improvement